scripts/database/restore.sh

```diff
@@ -30,27 +30,13 @@ else
   echo "Getting values from .env"
 fi
 
+
 # if database has already been migrated, exit
 if [[ "$STRAPI_DATABASE_MIGRATED" == "true" ]]; then
   echo "Strapi database has already been migrated"
   exit 0
 fi
 
-# Skip for local development
-if [ ! -f /.dockerenv ]; then
-  # Add current public IP to firewall exceptions 
-  my_public_ip="$(curl -s https://api.ipify.org)"
-
-  echo "Adding current public IP to firewall exceptions..."
-  az postgres flexible-server firewall-rule create \
-    --resource-group "rg-$AZURE_ENV_NAME" \
-    --name "$CMS_DATABASE_SERVER_NAME" \
-    --rule-name "AllowMyIP" \
-    --start-ip-address "$my_public_ip" \
-    --end-ip-address "$my_public_ip" \
-    --output none
-fi
-
 filename="${1:-}"
```

Comment:
why these changes in this file?

Reply:
Uhhh.... I just did git rebase from the main.... let me remove